SConstruct

```diff
@@ -10,21 +10,27 @@ AddOption('--timing', action='append_const', dest='cflags',
 def buildServer(env):
 
   env.Append(
-    CPPPATH= ['src', 'utils/include',
+    CPPPATH= ['src', 'include', 'src/vcl', 'utils/include',
               '/usr/include/jsoncpp/',
               os.path.join(env['INTEL_PATH'], 'pmgd/include'),
               os.path.join(env['INTEL_PATH'], 'pmgd/util'),
-              os.path.join(env['INTEL_PATH'], 'vcl/include'),
-              os.path.join(env['INTEL_PATH'], 'vcl/src'),
              ],
     LIBS = [ 'pmgd', 'pmgd-util',
              'jsoncpp', 'protobuf', 'tbb',
-             'vdms-utils', 'vcl', 'pthread',
```

Comment:
I understand that you do not want to make a library out of the vcl folder because no other application will link against it. But I think library is more of a self contained unit of one kind of functionality. Unless there is a performance reason not to, it might still be sensible to make a library and clean up this scons file. All the opencv stuff will get encapsulated there that way.

Reply:
Also, by that logic, we will have to refactor utils too.

Reply:
we will keep vcl as a module, but not as a separate library. utils is actually linked against more than one component, vcl is not.